### DISCUSSION TITLE
Font Awesome icon usage in markdown (PDF and HTML formats)

### DISCUSSION THREAD

**Discussion by fuhrmanator** [quarto-dev/quarto-cli#887] [Q&A] [answered]
I'm used to 'hacking' my way through pandoc to get things like Font Awesome to work, but I'm wondering what the clean way to do it in quarto is?

I discovered that the callouts in Quarto use font awesome icons, so I realized in my markdown I can get them to work by just citing the LaTeX package's usage, e.g., `\faBook` will give me the book icon in the PDF.

But how can I do this in Markdown so that the HTML format also shows the icon? I want to avoid embedding HTML if possible.

MM uses the `@icon-name` syntax, which looks pretty clean: https://markdownmonster.west-wind.com/docs/_4ue01xl6q.htm

Is there a similar clean way to use FA icons in markdown in Quarto?

**Comment by jjallaire** [comment, marked as answer]
We will fairly soon have an extension mechanism that will allow to you create a custom shortcode for icons (including carrying along the requisite HTML or TeX dependencies). The syntax might look something like this:

```
{{< icon fa-github >}}
```
